public/js/components/RightSidebar.js

```diff
@@ -98,19 +103,29 @@ const RightSidebar = React.createClass({
     this.setupKeyboardShortcuts();
   },
 
-  shouldComponentUpdate(nextProps) {
-    return this.props.keyShortcuts !== nextProps.keyShortcuts;
-  },
-
   renderStepButtons() {
+    const className = this.props.pause ? "active" : "disabled";
     return [
-      debugBtn(this.resume, "resume", "active", "Click to resume (F8)"),
-      debugBtn(this.stepOver, "stepOver", "active", "Step Over (F10)"),
-      debugBtn(this.stepIn, "stepIn", "active", "Step In (F11)"),
```

Comment:
As a general UX note, i think it would be nice if this were a toggle. The benefit would be that the user could disable this option once it is set.